src/deploy/osp_deployer/settings/sample_xsp_profile.ini

```diff
@@ -652,14 +652,72 @@ node_types=
 # section name the same as the node type name:
 # [edge-compute-denver]
 # cidr = 192.168.121.0/24
+# mgmt_cidr = 192.168.111.0/24
 # dhcp_start = 192.168.121.121
 # dhcp_end = 192.168.121.250
 # inspection_iprange = 192.168.121.51,192.168.121.120
 # gateway = 192.168.121.1
+# nic_port_count = 5
+# When using any 4 port NIC configuration, comment out or delete the
+# ProvisioningInterface line below.
+# ProvisioningInterface=em1
+# Bond0Interface1=p1p1
+# Bond0Interface2=p3p1
+# Bond1Interface1=p1p2
+# Bond1Interface2=p3p2
+# private_api_vlanid=141
+# private_api_network=192.168.141.0/24
+# private_api_gateway=192.168.141.1
+# private_api_allocation_pool_start=192.168.141.20
+# private_api_allocation_pool_end=192.168.141.60
+# tenant_network=192.168.131.0/24
+# tenant_gateway=192.168.131.1
+# tenant_vlanid=131
+# tenant_allocation_pool_start=192.168.131.20
+# tenant_allocation_pool_end=192.168.131.60
+# storage_network=192.168.171.0/24
+# storage_gateway=192.168.171.1
+# storage_vlanid=171
+# storage_allocation_pool_start=192.168.171.20
+# storage_allocation_pool_end=192.168.171.60
+# external_network=100.67.62.128/26
+# external_vlanid=623
+# external_gateway=100.67.62.129
+# external_allocation_pool_start=100.67.62.140
+# external_allocation_pool_end=100.67.62.150
 #
 # [edge-compute-boston]
 # cidr = 192.168.122.0/24
+# mgmt_cidr = 192.168.112.0/24
 # dhcp_start = 192.168.122.121
 # dhcp_end = 192.168.122.250
 # inspection_iprange = 192.168.122.51,192.168.122.120
 # gateway = 192.168.122.1
+# nic_port_count = 5
+# When using any 4 port NIC configuration, comment out or delete the
+# ProvisioningInterface line below.
+# ProvisioningInterface=em1
+# Bond0Interface1=p1p1
+# Bond0Interface2=p3p1
+# Bond1Interface1=p1p2
+# Bond1Interface2=p3p2
```

Comment:
we should probably use new nic format in samples ?

Reply:
Not for JS-13.3 branch.  Agree for JS-16.1 though

Reply:
16.1 will have correct interfaces